sdk/parents/azure-client-sdk-parent/pom.xml

```diff
@@ -1386,5 +1387,32 @@
         </plugins>
       </build>
     </profile>
+
+    <profile>
+      <id>native</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.graalvm.buildtools</groupId>
+            <artifactId>native-maven-plugin</artifactId>
+            <version>0.9.2</version>  <!-- {x-version-update;org.graalvm.buildtools:junit-platform-native;external_dependency} -->
+            <configuration>
+              <skip>${graalvm.enable}</skip>
+              <buildArgs>--no-fallback</buildArgs>
+            </configuration>
```

Comment:
What's the thinking here?

Reply:
This is to centralize the graalvm plugin configuration. Any client library that wants to enable graalvm can then simply set the property to true in its pom.xml similar to how [appconfig is setup](https://github.com/Azure/azure-sdk-for-java/pull/23623/files#diff-51612a94b6cb94dec13f98ccb5e9a3c629f4c411efb7f9e69eb78a85a429cd41R35-R37)